server/dummyData/dummySurveys.js

```diff
@@ -245,7 +245,7 @@ module.exports = [
     datePublished: 1574259113,
     dateToClose: 1576851113,
     dateClosed: 1576851113,
-    anonymous: true,
+    anonymous: false,
```

Comment:
why is this necessary?

Reply:
the survey says it is anonymous however responses are recorded with employee ids